src/tools/debugger/session/Breakpoints.java

```diff
@@ -195,6 +212,70 @@ public Breakpoint getAsyncMessageRcvBreakpoint(final URI sourceUri,
     return bp;
   }
 
+  public synchronized void addReceiverBreakpoint(final URI sourceUri, final int startLine,
+      final int startColumn, final int charLength) {
+    SectionBreakpoint bId = new SectionBreakpoint(sourceUri, startLine, startColumn, charLength);
```

Comment:
I think we can declare `BreakpointId bId = new SectionBreakpoint(sourceUri, startLine, startColumn, charLength);`
in order to use the abstract class